src/library_nodefs.js

```diff
@@ -203,7 +203,8 @@ mergeInto(LibraryManager.library, {
       readlink: function(node) {
         var path = NODEFS.realPath(node);
         try {
-          return fs.readlinkSync(path);
+          path = fs.readlinkSync(path);
+          return PATH.join(node.mount.mountpoint, path);
```

Comment:
When you mount a node filesystem under a mountpoint (say `/working`), the symlink gets resolved incorrectly. E.g.: `/working/Users/janjongboom/repos` resolves into `/Volumes/repos`, which is not a valid folder in the emscripten filesystem. On node.js, which deals with real files & real symlinks we need to assume that this symlink resolves to another folder/file on the same filesystem. Ergo: need to add the mountpoint to the result of readlink.